roles/kiwix/tasks/install.yml

```diff
@@ -110,10 +100,14 @@
     - { src: 'iiab-make-kiwix-lib', dest: '/usr/bin/iiab-make-kiwix-lib', mode: '0755' }
     - { src: 'iiab-make-kiwix-lib3.py', dest: '/usr/bin/iiab-make-kiwix-lib.py', mode: '0755' }
 
-- name: "Install from template: Apache's kiwix.conf"
+- name: daemon_reload, so systemd (re)reads kiwix-serve.service
+  systemd:
+    daemon_reload: yes
+
+- name: "Install from template: /etc/{{ apache_conf_dir }}/kiwix.conf"
   template:
     src: kiwix.conf.j2
-    dest: "/etc/{{ apache_conf_dir }}/kiwix.conf"
+    dest: "/etc/{{ apache_conf_dir }}/kiwix.conf"    # apache2/sites-available
```

Comment:
*Nitpick* Wish you stop polluting action lines with notes. Notes should be above or below the stanza to keep the code readable as the variables can be used in the 'name' line also as above.